index.d.ts

```diff
@@ -2,27 +2,30 @@ interface Options {
   spacer?: number;
   prefix?: string;
   logger?: (method: string, space: string, path: string) => void;
+  color?: boolean;
 }
 
 interface Route {
-  method?: string;
+  method?: string | object;
```

Comment:
@yutak23 Do we know the shape of the object for the method so we can tighten it up more? 

Looking at @types/express it just defines it as any :( 

Happy to leave as object but wanted to ask the question.

Reply:
@labithiotis 

> Do we know the shape of the object for the method so we can tighten it up more?

Yes, I think it is possible.
However, I think it would take a lot of effort to define it in interface and type, and to define the type for each version of Express.

Therefore, I adopted `object` this time as a compromise (because `any` would allow impossible things such as `number` and `boolean`).